src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs

```diff
@@ -11335,6 +11335,1929 @@ public override BoundNode VisitExpressionWithNullability(BoundExpressionWithNull
         }
     }
 
+    internal sealed partial class NullabilityRewriter : BoundTreeRewriter
+    {
+        private readonly ImmutableDictionary<BoundExpression, TypeSymbolWithAnnotations> _topLevelNullabilities;
```

Comment:
>TypeSymbolWithAnnotations> _topLevelNullabilities [](start = 62, length = 49)

Same here. Feels like the dictionary should hold `Nullability` values.

Reply:
Can't use Nullability here, that can only communicate top level nullability. Inference can cause the nested nullability to change, so we need the full TSWA.

---
In reply to: [251672541](https://github.com/dotnet/roslyn/pull/32899#discussion_r251672541) [](ancestors = 251672541)